docs/en/docs/ecosystem/udf/java-user-defined-function.md

```diff
@@ -140,23 +140,25 @@ public class SimpleDemo  {
     }
 
     /*required*/
-    public void serialize(State state, DataOutputStream out) throws Exception {
+    public void serialize(State state, DataOutputStream out)  {
```

Comment:
could throw exception, now have plan to handle exception in BE.

Reply:
I know, but currently, the exception handling hasn't been dealt with properly yet. I have crashed BE several times because of this. So, at this stage, I hope that users can use it, and for example, in the case of flink-udf, they also expect users to handle exceptions by themselves. If an exception is thrown, the task will fail.